src/Options.php

```diff
@@ -642,6 +632,42 @@ public function setBeforeBreadcrumbCallback(callable $callback): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * @param null|TransportInterface $transport
+     */
+    public function setTransport(?TransportInterface $transport): void
+    {
+        $options = array_merge($this->options, ['transport' => $transport]);
+
+        $this->options = $this->resolver->resolve($options);
+    }
+
+    /**
+     * @return null|TransportInterface
+     */
+    public function getTransport(): ?TransportInterface
```

Comment:
As said by @ste93cry, this is a dependency, not an option... It doesn't belong here